tests/integration/targets/meraki_switchport/tasks/main.yml

```diff
@@ -226,6 +226,29 @@
       - update_port_vvlan.data.voice_vlan == 11
       - update_port_vvlan.changed == True
 
+- name: Configure access port removing voice VLAN
```

Comment:
One more small change and I think we're there. The placement of this is breaking the next text. Please drop this test down one test. I think it should be line 252.